specification/hugr.md

```diff
@@ -1226,11 +1226,26 @@ nodes as children.
 
 ###### `OutlineCFG`
 
-Replace a CFG-convex subgraph (of sibling BasicBlock nodes) having a
-single entry node with a single BasicBlock node having a CFG node child
-which has as its children the original BasicBlock nodes and an exit node
-that has inputs coming from all edges of the original CFG that don’t
-terminate in it.
+Replace a set of CFG sibling nodes with a single BasicBlock node having a
+CFG node child which has as its children the set of BasicBlock nodes
+originally specified. The set of basic blocks must satisfy constraints:
+* There must be at most one node in the set with incoming (controlflow) edges
+ from nodes outside the set. Specifically,
+   * *either* the set includes the CFG's entry node, and any edges from outside
+     the set (there may be none or more) target said entry node;
+   * *or* the set does not include the CFG's entry node, but contains exactly one
+     node which is the target of edges from outside the set.
+* The set may not include the exit block.
+* There must be exactly one edge from a node in the set to a node outside it.
+
+Situations in which multiple nodes have edges leaving the set, can be converted to
+this form by a combination of InsertIdentity operations and one Replace. Similarly/for example,
```

Comment:
```suggestion
this form by a combination of InsertIdentity operations and one Replace. For example,
```

Reply:
Yes, I thought "Similarly/for example" was a bit of a stretch ;). I've removed similarly but extended the class of things (just before) to include the example :)